web/build.mjs

```diff
@@ -9,12 +10,21 @@ import { fileURLToPath } from "url";
 
 const __dirname = fileURLToPath(new URL(".", import.meta.url));
 
+// Use the package.json file in the root folder, as it has the current version information.
+const authentikProjectRoot = execFileSync("git", ["rev-parse", "--show-toplevel"], { encoding: "utf8" }).replace(
```

Comment:
I'm not sure if that'll work in the container build, since we don't have .git available there currently? (Although we might want to change that since I've also been meaning to try out https://github.com/codecov/codecov-javascript-bundler-plugins)

Reply:
That, plus we don't have git binaries either